### PR TITLE
Carry v0.4.3 master bump

### PR DESCRIPTION
**Pull request by riyazdf**
Carries patches from v0.4.3 back to master - the delegation key import panic fix and netgo flag for server flag panic.

There was some messiness because of repeat lint commits that already made it to master, so I removed those in this PR.

### CODE REVIEW

File: buildscripts/cross.sh [hunk 1]
```diff
@@ -40,8 +40,8 @@ for os in "$@"; do
 	go build \
 		-o "${NOTARYDIR}/cross/${GOOS}/${GOARCH}/${OUTFILE}" \
 		-a \
-		-tags "${BUILDTAGS}" \
+		-tags "${BUILDTAGS} netgo" \
 		-ldflags "-w ${CTIMEVAR} ${LDFLAGS}"  \
 		./cmd/notary;
 	set +x;
-done
```
Comment (cyli) [inline buildscripts/cross.sh]: Non-blocking nitpicky-ness - could we add a newline afterward?

Comment (riyazdf) [inline buildscripts/cross.sh]: done!

**Comment by endophage** [comment]
@riyazdf can you rebase then assuming tests pass we're good to merge.

**Comment by riyazdf** [comment]
green after rebase!  Merging.

**Pull request merged**